### PR TITLE
updating chess readme

### PR DESCRIPTION
**Pull request by pdg744**
In #683, we updated the `bevy` README with the following organization: 

`Introduction`
`Quick Start`
`Use Cases`
`Project Organization`
`More Resources`

This PR moves the links into each section to facilitate more modular re-use of content, and revises the `chess` README with an analogous organization. 

The `chess` README also includes an `Approach` section; not sure we came to clarity on whether we want to always/sometimes/never include this sort of section. My instinct is sometimes.

### CODE REVIEW

**Review by tzerrell — COMMENTED** [review]
Thanks!

File: examples/bevy/README.md [hunk 1]
```diff
@@ -1,32 +1,37 @@
...
 The receipt can be passed to a third party, who can examine the [journal] to check the program's outputs and can [verify] the [receipt] to ensure the integrity of the [guest program]'s execution.
 
-## More Resources
-For more information about building, running, and testing zkVM applications, see our [developer docs].
-
-[bevy]: https://bevyengine.org/
-[install Rust]: https://doc.rust-lang.org/cargo/getting-started/installation.html
-[zkVM]: https://dev.risczero.com/terminology#zero-knowledge-virtual-machine-zkvm
+[`src/main.rs`]: /src/main.rs
+[`methods/guest/src/main.rs`]: methods/guest/src/main.rs
```
[the remark this answers is not in the document]
Comment (pdg744) [reply examples/bevy/README.md:34]: I don't have a strong opinion here -- it's easy enough to switch to the full paths if that seems better.

[the remark this answers is not in the document]
Comment (tzerrell) [reply examples/bevy/README.md:34]: Switch to full path then probably? But I don't think it matters much either way, probably most people clicking links are on GitHub.

**Review by tzerrell — APPROVED** [review]

File: examples/chess/README.md [hunk 2]
```diff
@@ -1,23 +1,76 @@
...
-## Quick Start
+[zkVM]: https://dev.risczero.com/zkvm
+[shakmaty]: https://docs.rs/shakmaty/latest/shakmaty/
 
-First, make sure [rustup](https://rustup.rs) is installed. This project uses a [nightly](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) version of [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html). The [`rust-toolchain`](rust-toolchain) file will be used by `cargo` to automatically install the correct version.
 
-To build all methods and create a zero-knowledge proof that the position defined [here](https://github.com/risc0/risc0/blob/main/examples/chess/src/main.rs#L28) includes a mate-in-one, run the following command from this directory:
+## Quick Start
 
+First, [install Rust] if you don't already have it.
```
Comment (pdg744) [inline examples/chess/README.md:13]: Oh; just realized that this drops the notes about `nightly` and `rust-toolchain`, which we probably want to keep (and should add to `bevy`).

Comment (pdg744) [reply examples/chess/README.md:13]: And maybe we want to be encouraging `rustup` still?

Comment (flaub) [reply examples/chess/README.md:13]: We will need to have:
```
cargo install cargo-risczero
cargo risczero install
```

**Pull request merged**